cmd/kyma/alpha/list/module/module.go

```diff
@@ -88,8 +95,8 @@ List all modules for the kyma "some-kyma" in the "alpha" channel
 		&o.Timeout, "timeout", "t", 1*time.Minute, "Maximum time for the list operation to retrieve ModuleTemplates.",
 	)
 	cmd.Flags().StringVarP(
-		&o.Namespace, "namespace", "n", metav1.NamespaceDefault,
-		"The namespace to use. An empty namespace uses 'default'",
+		&o.Namespace, "namespace", "n", "kyma-system",
+		"The namespace to list modules in. An empty namespace uses 'kyma-system'",
```

Comment:
The message already contains hint for the default value.
```suggestion
		"The namespace to list modules in",
```